openapi/openapi/src/main/scala/endpoints4s/openapi/JsonSchemas.scala

```diff
@@ -116,13 +116,19 @@ trait JsonSchemas extends algebra.JsonSchemas with TuplesSchemas {
 
     // A documented JSON schema that is unevaluated unless its `value` is accessed
     sealed abstract class LazySchema extends DocumentedJsonSchema {
+      def name: String =
+        sys.error(s"Unsupported algebra version: 2.1.0. Please update your interpreter.")
```

Comment:
In my opinion, it’s fine to leave it abstract and to bump the major version number.

It’s important to keep binary compatibility as long as possible in the algebra modules, because in a microservice architecture you may deal with APIs of multiple (independent) services, each using a different version of the algebra of endpoints4s, so we want them to be all binary compatible. However, dependencies on interpreters (such as openapi) are typically direct dependencies (as opposed to transitive dependencies), so it is less important to keep them binary compatible.

Reply:
Sounds, good. To be honest, I'm not very used to these binary compatibility considerations, I just try to get the github action happy 😊